### DISCUSSION TITLE
Importing excel [Aug 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3126] [Discord Threads]
> 🙋‍♂️ **Amin Debabeche**  
## Hi I have an issue while importing an excel, could someone help on this ? `undefined is not an object (evaluating 'e.indexOf')`  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1004686879221887016/MicrosoftTeams-image_3.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Within excel, each sheet has to have a header column which gets translated to column names   
####    
#### and if you can ensure all column are rightly mapped it will help  
#  
> 🙋‍♂️ **Amin Debabeche**  
#### What do you mean by rightly mapped ?  
#  
> 💁 **o1lab**  
#### we automap columns to a column type  
#  
> 💁 **o1lab**  
#### its only best effort   
#### sometimes it may not be right  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1004686879892967474)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
